plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/ExasolQueryRunner.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exasol;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.trino.Session;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.MaterializedResult;
+import io.trino.tpch.TpchTable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.trino.plugin.exasol.ExasolTpchTables.copyAndIngestTpchData;
+import static io.trino.plugin.exasol.TestingExasolServer.TEST_SCHEMA;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public final class ExasolQueryRunner
+{
+    private static final Logger log = Logger.get(ExasolQueryRunner.class);
+
+    private ExasolQueryRunner() {}
+
+    public static DistributedQueryRunner createExasolQueryRunner(
```

Comment:
Please use a builder pattern. You can refer to other connectors, e.g. `IgniteQueryRunner`.